pkg/metrics/operator_prom_metrics.go

```diff
@@ -30,10 +30,20 @@ var (
 		},
 		[]string{"type"},
 	)
+
+	crdTotalsGaugeVec = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "keda_operator",
+			Subsystem: "cr",
```

Comment:
Here I am again to be a pain but `cr` is really a bad name.

Can we please rename it to `keda_operator_resource_totals` or `keda_operator_customresource_totals` please?

```suggestion
			Subsystem: "resource",
```
That way people know what it means when not looking at the doc page